dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -3550,6 +3550,35 @@ ReturnedColumn* buildReturnedColumn(Item* item, gp_walk_info& gwi, bool& nonSupp
   return rc;
 }
 
+// parse the boolean fields to string "true" or "false"
+ReturnedColumn* buildBooleanConstantColumn(Item* item, gp_walk_info& gwi, bool& nonSupport)
+{
+  ConstantColumn* cc = NULL;
+
+  if (gwi.thd)
+  {
+    {
+      if (!item->fixed())
+      {
+        item->fix_fields(gwi.thd, (Item**)&item);
+      }
+    }
+  }
+  cc = new ConstantColumnSInt(colType_MysqlToIDB(item), (int64_t)item->val_int() ? "true" : "false",
```

Comment:
I might better use static_cast and put item->val_int() into a variable to reduce number of calls.

Reply:
it Looks good! I have fixed it.